hail_search/test_search.py

```diff
@@ -341,6 +348,63 @@ async def test_multi_project_search(self):
             sample_data={**MULTI_PROJECT_SAMPLE_DATA, **SV_WGS_SAMPLE_DATA},
         )
 
+    async def test_both_sample_types_search(self):
+        # One family (F000011_11) in a multi-project search has identical exome and genome data.
+        expected_variants = [PROJECT_2_VARIANT, MULTI_PROJECT_VARIANT1, MULTI_PROJECT_VARIANT2, VARIANT3, VARIANT4]
+        expected_results = []
+        for variant in expected_variants:
+            v = deepcopy(variant)
+            for indiv_id, gts in v['genotypes'].items():
+                v['genotypes'][indiv_id] = [gts]
+            if 'I000015_na20885' in v['genotypes']:
+                v['genotypes']['I000015_na20885'].append({**v['genotypes']['I000015_na20885'][0], 'sampleType': 'WGS'})
+            expected_results.append(v)
+
+        await self._assert_expected_search(
+            expected_results, gene_counts=GENE_COUNTS, sample_data=MULTI_PROJECT_SAMPLE_TYPES_SAMPLE_DATA,
+        )
+
+        # Variant in family_4 is de novo in exome but maternally inherited in genome.
+        # Genome passes quality and inheritance, show genotypes for both sample types.
```

Comment:
I think these tests would be more maintainable if we used the existing families. Since we do not have WGS data for family 2 you could add a WGS table for that family with variants that cover you edge cases with the otherwise existing fixtures. `VARIANT1` is already de novo in exome, so adding the same variant as maternally inherited in genome seems easier that adding and remembering about these new variants